### PR TITLE
Feature/gsdavies stdana dev

### PR DESCRIPTION
**Pull request by gavinsdavies**
Prepping a pull request for introducing `CAFMaker` and `StandardRecord` so that one can produce CAFs and perform ntuple-style analysis; ease of analysis at the command line.
Includes updates to comply with `cetmodules v3.20` though more are needed, grabbing latest `srproxy v00.42` for support making standard records.
I think I need to add OnMon back into this build. Will checkbefore finalizing PR.
Draft PR simply because I'm first checking the full chain works to produce plots; it builds and produces a caf file. I just need to do a couple more cross-checks. Expect full PR tomorrow.

### CODE REVIEW

File: StandardRecord/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		  LIBRARIES    ${ROOT_BASIC_LIB_LIST}
+		  	       ${ROOT_CORE}
+			       ${ROOT_LIBRARIES}
+		)
+
+build_dictionary( StandardRecord
+		  DICTIONARY_LIBRARIES StandardRecord
+		)
+
+#build_dictionary( StandardRecord
```
Comment (lackey32) [inline StandardRecord/CMakeLists.txt:29]: Is there a reason to keep this comment block?

File: StandardRecord/SRVector3D.cxx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRVector3D.cxx
+// \author  Christopher Backhouse - bckhouse@caltech.edu
```
Comment (lackey32) [inline StandardRecord/SRVector3D.cxx:3]: Can we remove the author line here?

File: StandardRecord/SRVector3D.h [hunk 3]
```diff
@@ -0,0 +1,67 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRVector3D.h
+// \author  Christopher Backhouse - bckhouse@caltech.edu
```
Comment (lackey32) [inline StandardRecord/SRVector3D.h:3]: Remove author line.

**Review by lackey32 — APPROVED** [review]
A couple minor comments, but I'm pretty sure they only relate to commented out lines. Assuming this still builds and run as expected, I approve.

File: StandardRecord/CMakeLists.txt (hunk 1 above)
Comment (gavinsdavies) [reply StandardRecord/CMakeLists.txt]: removed.

File: StandardRecord/SRVector3D.cxx (hunk 2 above)
Comment (gavinsdavies) [reply StandardRecord/SRVector3D.cxx]: yes, removed

File: StandardRecord/SRVector3D.h (hunk 3 above)
Comment (gavinsdavies) [reply StandardRecord/SRVector3D.h]: yes, removed

**Pull request merged**